software_training/src/action.cpp

```diff
@@ -17,15 +17,16 @@ int main (int argc, char **argv)
        = nh.advertise<geometry_msgs::Twist>("/moving_turtle/cmd_vel", 1000);
   // create the action client
   // true causes the client to spin its own thread
-
-  actionlib::SimpleActionClient<goemetry_msgs::Twist>("action", true);
+  
```

Comment:
sorry forgot to mention this, action Client and Server should never be in the same node, it defeats the purpose of having an action server.